libeos-payg/tests/manager.c

```diff
@@ -612,18 +620,20 @@ test_manager_error_malformed (Fixture *fixture,
 {
   manager_new (fixture);
   g_autoptr(GError) error = NULL;
+  gint64 time_added = NULL;
   guint64 expiry, expiry_after_code;
   gboolean ret;
 
   expiry = epg_provider_get_expiry_time (fixture->provider);
 
   /* Just use an obviously-wrong code. The nuances are tested in libeos-payg-codes. */
-  ret = epg_provider_add_code (fixture->provider, "abcdefgh", &error);
+  ret = epg_provider_add_code (fixture->provider, "abcdefgh", &time_added, &error);
   g_assert_error (error, EPG_MANAGER_ERROR, EPG_MANAGER_ERROR_INVALID_CODE);
   g_assert_false (ret);
 
   expiry_after_code = epg_provider_get_expiry_time (fixture->provider);
   g_assert_cmpint (expiry, ==, expiry_after_code);
+  g_assert_null (time_added);
```

Comment:
`time_added` is not a pointer. I think you want `g_assert_cmpint (time_added, ==, 0)` (or no assertion).

Reply:
Is `-Wint-conversion` not enabled, or is someone ignoring warnings?

Reply:
I'm not used to working with it, so it does seem that i've been summarily ignoring warnings...

Reply:
Oh, yeah, it's giving warnings because I end up comparing guint with gint - even though these two values are comparable it does break when we're assuming the user added a infinite code value.